scripts/codecov.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash
+FILENAME=codecov
+curl -s https://codecov.io/bash > $FILENAME
+CODECOV_VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
+VALIDATION_FAILURE=0
+for i in 1 256 512
+do
+  IS_DIFF=$(diff <(shasum -a $i $FILENAME) <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/$CODECOV_VERSION/SHA${i}SUM))
```

Comment:
This would fail as the output is including also the shasum of the env file.
As far as I can tell you are not using this file, then drop the second line with a "head -n 1" command.

Reply:
Good point
Done within https://github.com/pagopa/io-app/pull/2988/commits/c21b0d14984d96d3f57529c17d2be573a27c9533